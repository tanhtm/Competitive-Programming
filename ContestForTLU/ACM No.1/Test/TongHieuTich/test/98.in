10
853 24
-295 407
151 165
302 -213
549 -294
727 159
-558 47
-25 -455
-478 -435
-193 573
