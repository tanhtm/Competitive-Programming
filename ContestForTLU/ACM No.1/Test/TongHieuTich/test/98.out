181758
-19656
-1106
11458
53741
125812
77288
-51600
9814
-72770
